src/test/java/org/incodelearning/algorithm/LinkedListTest.java

```diff
@@ -0,0 +1,48 @@
+package org.incodelearning.algorithm;
+
+import org.junit.Test;
+
+import java.util.LinkedList;
+import java.util.Queue;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Methods to use LinkedList (Java's LinkedList is doubly linked list) as stack and queue.
```

Comment:
For this topic, you may compare and summarize LinkedList, ArrayList, and ArrayDeque.